### PR TITLE
fix: Bump realm-gradle-plugin to 6.1.0 and fix build

### LINKED ISSUE #2473: Bump realm-gradle-plugin to 6.1.0 and fix build
**Describe the chore**
Regarding #2380 

**Would you like to work on the issue?**
  Yes

### PR DESCRIPTION
**Pull request by rob729**
Fixes #2473, #2380

### CODE REVIEW

**Comment by rob729** [comment]
@atm1504 @iamareebjamal Can you please re-trigger the ci build as I check for any possible reason for build fail but found none?

**Comment by liveHarshit** [comment]
Where is Travis

File: build.gradle [hunk 1]
```diff
@@ -19,6 +19,7 @@ buildscript {
 
 allprojects {
     repositories {
+        jcenter()
```
Comment (atm1504) [inline build.gradle]: Why did u add jcenter().

Comment (rob729) [reply build.gradle]: Because the build was failing and it was showing 7 libraries as failed to resolve.

**Comment by rob729** [comment]
I don't think susi ai android uses Travis.

> @atm1504 @iamareebjamal Can you please re-trigger the ci build as I check for any possible reason for build fail but found none?

**Comment by atm1504** [comment]
> I don't think susi ai android uses Travis.
> 
> > @atm1504 @iamareebjamal Can you please re-trigger the ci build as I check for any possible reason for build fail but found none?

Susi uses circle ci yml file not travis

File: build.gradle (hunk 1 above)
Comment (rob729) [reply build.gradle]: @atm1504  can you please re-trigger the build or help me identify the reason for build fail error?

**Comment by atm1504** [comment]
@rob729 check this out https://circleci.com/gh/fossasia/susi_android/5837?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by iamareebjamal** [comment]
When compilation fails, do you just retry compiling again or try to fix the error?

**Comment by rob729** [comment]
The build error is fixed now. Please review the PR.

**Pull request merged**